static/app/views/profiling/landing/table.tsx

```diff
@@ -0,0 +1,152 @@
+import {useMemo} from 'react';
+import {Location} from 'history';
+
+import DateTime from 'sentry/components/dateTime';
+import Duration from 'sentry/components/duration';
+import GridEditable, {
+  COL_WIDTH_UNDEFINED,
+  GridColumnOrder,
+} from 'sentry/components/gridEditable';
+import {IconCheckmark, IconClose} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {Trace} from 'sentry/types/profiling/trace';
+import {Container, NumberContainer} from 'sentry/utils/discover/styles';
+
+interface Props {
+  location: Location;
+  traces: Trace[];
+}
+
+function ProfileTable({location, traces}: Props) {
+  const data: TableDataRow[] = useMemo(
+    () =>
+      traces.map(trace => ({
+        failed: Boolean(trace.failed), // makes this a required field
+        ...trace,
+      })),
+    [traces]
+  );
+
+  return (
+    <GridEditable
+      isLoading={false}
+      data={data}
+      columnOrder={COLUMN_ORDER.map(key => COLUMNS[key])}
+      columnSortBy={[]}
+      grid={{renderBodyCell}}
+      location={location}
+    />
+  );
+}
+
+type TableColumnKey =
```

Comment:
`Pick<Profiling.Trace, id | ....>` because there's some relationship between the two and is more future proof to changes.  e.g. if we change Trace type, we can avoid having the table render empty because we forgot to update the keys or something